models/apartment.rb

```diff
@@ -0,0 +1,25 @@
+require 'pry'
+
+class Apartment < ActiveRecord::Base
+
+  has_many :tenants
+
+  def initialize address, monthly_rent, sqft, num_beds, num_baths
+    @address = address.to_s
+    @monthly_rent = monthly_rent
+    @sqft = sqft
+    @num_beds = num_beds
+    @num_baths = num_baths
+    @renters = []
+  end
```

Comment:
Since you're using ActiveRecord, you actually don't need an initialize method anymore.